backend/service/services.go

```diff
@@ -64,8 +69,18 @@ func InitStatsD(statsdURL string, log *logrus.Logger) *statsd.Client {
 	return client
 }
 
-func InitDB(databaseURL string, log *logrus.Logger) *pgx.Conn {
-	connection, err := pgx.Connect(context.Background(), databaseURL)
+func InitPgDB(pgdbURL string, log *logrus.Logger) *pgx.Conn {
+	connection, err := pgx.Connect(context.Background(), pgdbURL)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return connection
+}
+
+func InitMsDB(msdbURL string, log *logrus.Logger) *mssql.DB {
+	// connection string with placeholders user, password, database
+	connString := "server=" + msdbURL + ";user id=user;password=password;database=database"
+	connection, err := mssql.Connect(context.Background(), connString)
 	if err != nil {
 		log.Fatal(err)
 	}
```

Comment:
```suggestion
	connector, err := mssql.NewConnector(msdbURL)
	if err != nil {
		log.Fatal(err)
	}

	connection, err := connector.Connect(context.Background())
	if err != nil {
		log.Fatal(err)
	}

	return &connection
```

Reply:
and delete [this line](https://github.com/elonsoc/ods/blob/a0b11b1451069d04f66bf3401662214f9f83e584/backend/service/services.go#L93) since you're returning a pointer.